src/utils/work/seekWorkResponse.js

```diff
@@ -0,0 +1,101 @@
+import moment from 'moment';
+import Amplify, { Storage } from 'aws-amplify';
+
+import connectionPromise from '../socketConnection';
+import WorkResponseError from '../WorkResponseError';
+import getAuthJWT from '../getAuthJWT';
+import WorkTimeoutError from '../WorkTimeoutError';
+
+const seekFromS3 = async (ETag) => {
+  const configuredBucket = Amplify.configure().Storage.AWSS3.bucket;
+  const storageUrl = await Storage.get(
+    ETag,
+    {
+      bucket: configuredBucket.replace('biomage-originals', 'worker-results'),
```

Comment:
Can we have a variable in config for the work results bucket and then just use it here instead of this hardcoding?

Reply:
we can't, as the `configBucket` is derived from AWS so we can't access it from the browswer